modules/reference/pages/public-metrics-reference.adoc

```diff
@@ -668,6 +668,84 @@ Total number of requests that uploaded an object to cloud storage.
 
 ---
 
+[[tls_metrics]]
+== TLS metrics
+
+=== redpanda_tls_truststore_expires_at_timestamp_seconds
+
+The expiration time, represented as a unix-style timestamp, of the shortest-lived CA in the installed CA chain. This lists all resources that have at least one CA configured.
+
+*Type*: gauge
+
+*Labels*:
+
+* `area`
+* `detail`
+
+*Usage*: This metric is recorded for all resources that may have a TLS certificate installed. The `area` and `detail` labels provide a hierarchical way of identifying a single resource you need information on.
+
+---
+
+=== redpanda_tls_certificate_expires_at_timestamp_seconds
+
+The expiration time, represented as a unix-style timestamp, for the shortest-lived installed certificate. While up to five certificates may be installed for a resource, this provides only the timestamp of the next to expire. This lists all resources that have at least one certificate installed.
+
+*Type*: gauge
+
+*Labels*:
+
+* `area`
+* `detail`
+
+*Usage*: This metric is recorded for all resources that may have a TLS certificate installed. The `area` and `detail` labels provide a hierarchical way of identifying a single resource you need information on.
```

Comment:
I noticed that both the Usage notes and the metric description both say something along the lines of "this will include all resources that have may have a certificate installed." Would it make sense to keep it in one part and omit it from the other?

Reply:
Copy+paste oversight, I'll rephrase, thanks!